integration/spark2/src/test/scala/org/apache/spark/carbondata/restructure/AlterTableRevertTestCase.scala

```diff
@@ -95,6 +96,17 @@ class AlterTableRevertTestCase extends QueryTest with BeforeAndAfterAll {
     }
   }
 
+  test("test to check if exception during rename table does not throws table not found exception") {
+    val locks = AlterTableUtil
+      .validateTableAndAcquireLock("default", "reverttest", List("meta.lock"))(sqlContext
+        .sparkSession)
+    val exception = intercept[RuntimeException] {
+      sql("alter table reverttest rename to revert")
+    }
+    assert(exception.getMessage == "Alter table rename table operation failed: Table is locked for updation. Please try after some time")
+    AlterTableUtil.releaseLocks(locks)
```

Comment:
release lock should be above assert. If testcase fails lock will not be released